portal/catalog.ipynb

```diff
@@ -0,0 +1,2615 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: \"Catalog\"\n",
+    "format:\n",
+    "    html:\n",
+    "        toc: true\n",
+    "        theme: darkly\n",
+    "        page-layout: full\n",
+    "        anchor-sections: true\n",
```

Comment:
We could force Quarto to run this notebook when rendering with this:

```
execute: 
  enabled: true
```

https://quarto.org/docs/tools/jupyter-lab.html#cell-execution

Reply:
Done.